quickboosters/api/users/dev/sample_data.py

```diff
@@ -16,23 +16,23 @@ def create_user():
     except Exception as e:
         print(str(e))
 
-def create_order():
-  order = Orders(order_type="solo_boost",user_id='1',order_amount=69.42,status='completed',date_ordered=datetime.datetime.now(),game='league of legends')
-  print(order.user_id)
-  db.session.add(order)
-  db.session.commit()
+#def create_order():
+  #order = Orders(order_type="solo_boost",user_id='1',order_amount=69.42,status='completed',date_ordered=datetime.datetime.now(),game='league of legends')
+  #print(order.user_id)
+  #db.session.add(order)
+  #db.session.commit()
 
 def Chatlog():
```

Comment:
chatlog function doesn't need to be in the sample_data function under users.

Reply:
Also, the create_order() function doesn't need to be in users/dev/sample_data.py even if it's commented out.